source/administration/command-line-tools.rst

```diff
@@ -655,3 +660,181 @@ platform version
     .. code-block:: none
 
       platform version
+
+Mattermost 3.5 and earlier
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Typing ``platform -help`` brings up documentation for the CLI tool.
+
```

Comment:
I think adding it only to the examples would be good, plus something to the intro section.